Resources/Locale/en-US/_CD/traits/traits.ftl

```diff
@@ -0,0 +1,2 @@
+trait-synth-name = Synthetic
+trait-synth-desc = You are a biomechanical construct, who bleeds coolant and is notified of ongoing Ion Storms.
```

Comment:
Bleeds coolant but the synth blood is described as not coolant
What is the truth???? /lh